apis/python/src/tiledbsoma/_experiment.py

```diff
@@ -22,8 +22,12 @@ class Experiment(
         AnyTileDBObject,
     ],
 ):
-    """An :class:`Experiment` represents a single-cell experiment. It is
-    a container class that has observations and measurements.
+    """A collection subtype representing an annotated 2D matrix of measurements.
```

Comment:
not sure how this is a 2D matrix -- `exp.ms` is a collection (1D) of measurements

Reply:
Agreed, I think it might be more accurate to combine the old and new descriptions. 
```suggestion
    """An :class: `Experiment` is a collection subtype that contains the observations and measurements from an individual experiment.
```